fvm/scriptEnv.go

```diff
@@ -685,3 +686,35 @@ func (e *ScriptEnv) Commit() ([]programs.ContractUpdateKey, error) {
 func (e *ScriptEnv) AllocateStorageIndex(_ []byte) (uint64, error) {
 	return 0, errors.NewOperationNotSupportedError("AllocateStorageIndex")
 }
+
+func (e *ScriptEnv) BLSVerifyPOP(pk *runtime.PublicKey, sig []byte) (bool, error) {
+	key, err := crypto.DecodePublicKey(crypto.BLSBLS12381, pk.PublicKey)
+	if err != nil {
+		return false, err
+	}
+	return crypto.VerifyPOP(key, sig)
```

Comment:
Same here, we probably should check for the sentinel error (we can also make the check under fvm/crypto)